build-tools/makefile_components/base_container_arm64.mak

```diff
@@ -0,0 +1,28 @@
+# Container section of standard makefile (arm64)
+##### PLEASE DO NOT CHANGE THIS FILE #####
```

Comment:
I'm removing the build-tools include from the main ddev Makefile, and consolidating needed parts of build-tools/makefile_components into that Makefile. You're welcome to do that here if it makes things easier, but at some point we have to have this logic in the proper Makefile, and of course, it's not legit to edit build_tools/makefile_components as long as we're using build-tools, since it's an upstream repo. Anyway, you can either continue this direction and I'll do a PR to remove build-tools or if it's easier you can consolidate yourself.